SC_redesign/Crop_and_Soil/crop/water_uptake.py

```diff
@@ -196,7 +196,7 @@ def _correct_layer_for_efficiency(potential_uptake: float, available_water: floa
     @staticmethod
     def _adjust_water_uptakes(potential_uptakes: List[float], unmet_demands: List[float],
                               uptake_compensation: float) -> List[float]:
-        """adjusts the potential water uptakes from each layer based by drawing from deeper layeres when possible.
+        """adjusts the potential water uptakes from each layer based by drawing from deeper layers when possible.
```

Comment:
```suggestion
        """Adjusts the potential water uptakes from each layer based by drawing from deeper layers when possible.
```

Reply:
This sentence is still a little confusingly worded. I think it's something in the "...from each layer based by drawing from..." construction. 

```suggestion
        """Adjusts the potential water uptakes from each layer by drawing from deeper layers when possible.
```

Are there multiple potential water uptakes for each layer?

Reply:
Honestly, I'm not sure either. I did not write that, let me look into it.

Reply:
Now I looked into it. No, there's only 1 uptake for each layer.